tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
Please do update the documentation to be correct :-)

As long as the combination of those rules (jsx-space-before-closing set to never, and this one) yields code that passes the linter, I think this works great :-)